packages/phoenix-evals/src/phoenix/evals/models/litellm.py

```diff
@@ -56,7 +56,7 @@ def _migrate_model_name(self) -> None:
     def _init_environment(self) -> None:
         try:
             import litellm
-            from litellm import validate_environment
+            from litellm.utils import validate_environment
```

Comment:
This part of this PR?

Reply:
yeah, I think at some point there was a refactor we need this to pass CI on evals

Reply:
shouldn't this go to main tho?